sdk/monitor/monitor-opentelemetry-exporter/src/utils/logUtils.ts

```diff
@@ -45,6 +45,11 @@ export function logToEnvelope(log: ReadableLogRecord, ikey: string): Envelope |
   const sampleRate = 100;
   const instrumentationKey = ikey;
   const tags = createTagsFromLog(log);
+  if (log.attributes[KnownContextTagKeys.AiOperationName]) {
```

Comment:
You can add this logic in createTagsFromLog, to maintain all tags logic in there